vector/src/main/java/im/vector/app/features/crypto/recover/BootstrapBottomSheet.kt

```diff
@@ -64,6 +68,25 @@ class BootstrapBottomSheet : VectorBaseBottomSheetDialogFragment<BottomSheetBoot
         return BottomSheetBootstrapBinding.inflate(inflater, container, false)
     }
 
+    private val reAuthActivityResultLauncher = registerStartForActivityResult { activityResult ->
+        if (activityResult.resultCode == Activity.RESULT_OK) {
+            when (activityResult.data?.extras?.getString(ReAuthActivity.RESULT_FLOW_TYPE)) {
```

Comment:
Generally I prefer to keep the key for bundles private to the Activity and provide fun in the companion to get the result, which could be a sealed class for this case.

Reply:
To be reworked later